app/views/provider/admin/cms/templates/update.js.erb

```diff
@@ -13,6 +13,11 @@ $('.inline-errors').remove();
     codemirror.save()
   }
 
+  // Remove error styles if any
+  $('#edit_cms_template').removeClass('error');
```

Comment:
Why this? :thinking:

Reply:
because this error class still presist there if page is valid then too

Reply:
Hm, from my tests the problem is that the `error` class persists on the inputs themselves, not on the form.